pkg/util/sys/device.go

```diff
@@ -472,3 +472,37 @@ func ListDevicesChild(executor exec.Executor, device string) ([]string, error) {
 
 	return strings.Split(childListRaw, "\n"), nil
 }
+
+// PartitionIsAtari returns true if the device partition is an Atari type.
+// Ceph bluestore raw disks can sometimes appear as though they have ATARI (AHDI)
+// partitions created on them when they don't in reality.
+// See: https://github.com/rook/rook/issues/7940
+func PartitionIsAtari(executor exec.Executor, device string) (bool, error) {
+	raw, cmdErr := executor.ExecuteCommandWithOutput("parted", "--machine", "--script", path.Join("/dev", device), "print")
```

Comment:
```suggestion
	raw, err := executor.ExecuteCommandWithOutput("parted", "--machine", "--script", path.Join("/dev", device), "print")
```

Why. not calling this like a common error?

Reply:
We keep it for a long time since the error doesn't always result in bailing out of the function. It seemed best to give it a unique name.

Reply:
`parted` throws an error when the partition type is unknown. This happens when there is no partition label on the partition and is a requirement currently because of the `ceph-volume inspect` command. So in the most-common condition, the command will always return an error.